src/bindings/writeTypescript.ts

```diff
@@ -217,6 +231,23 @@ export function writeTypescript(
         }
     });
     w.append(`]`);
+
+    //Opcodes
+    //So user can use them in sandbox tests
+    w.append("export abstract class Opcodes {");
```

Comment:
TypeScript has `enum` syntax for this. `abstract class` implies user should inherit from it to get access to its fields, and that's definitely not the way in a language without multiple inheritance.